docs/source/index.rst

```diff
@@ -112,6 +112,7 @@ Contents
    local-cluster
    ipython
    Joblib Integration <https://ml.dask.org/joblib.html>
+   monitoring-dask-cluster
```

Comment:
Same here.  Given that this document currently only discusses Prometheus, I would prefer that it was named `prometheus`.